staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/mutating/dispatcher.go

```diff
@@ -69,6 +69,9 @@ func (a *mutatingDispatcher) Dispatch(ctx context.Context, attr *generic.Version
 			}
 			glog.Warningf("Failed calling webhook, failing closed %v: %v", hook.Name, err)
 		}
+		if statusError, ok := err.(*apierrors.StatusError); ok {
```

Comment:
This makes sure the remote http status code could be sent to the client.

https://github.com/kubernetes/kubernetes/pull/64827/files#diff-e057ec5b2ec27b4ba1e1a3915f715262R100